SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.cxx

```diff
@@ -209,6 +209,7 @@ const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_AUTH_SERVER_INTEREST =
 
 	// Player controller
 	PLAYER_CONTROLLER_COMPONENT_ID,
+	PARTITION_COMPONENT_ID,
```

Comment:
Interesting, I guess we don't get authority over this component, and thus there is nothing trying to keep it in our view?
Still a bit confused why this sometimes worked and sometimes not

Reply:
Yep that's correct